superset/contrib/docker/superset_config.py

```diff
@@ -1,58 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
maybe we should move our config here too?

Reply:
I was going to say no because right now `superset_config` and `superset_config_dev` are in the same dir. But moving it here allows us to detect if there's anything added in upstream, so makes sense. Done in df208b52.